bash/solutii/micu_matei/man.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Scrieti un script pentru cauta informatii in manualul unei comenzi
+# ./man.sh nume_script informatie
+if [[ $# -lt 2 ]]; then
+    echo "Insuficienti parametri "
+    exit
```

Comment:
Iti recomand sa pui ` exit 1; `. Daca acest script este apelat si vrei sa folosesti exit_code, in loc de eroare(1) o sa iti apare succes(0).